src/Query/OrderField.php

```diff
@@ -12,8 +12,8 @@ class OrderField
     /**
      * Direction constants
      */
-    const ASCENDING = 0;
-    const DESCENDING = 1;
+    const int ASCENDING = 0;
+    const int DESCENDING = 1;
```

Comment:
I think this would be better as an enum. The value doesn't actually mean anything useful.